LocalPackages/pve-only/place.msa

```diff
@@ -153,3 +246,67 @@ nerdch.level.admin:/delplace $place = >>>
 		msg(color(LIGHT_PURPLE).$place . ' not found!')
 	}
 <<<
+
+nerdch.level.admin:/addendportal $endportal = >>>
+	@endportals = get_value('endportals')
+	if(@endportals == null){
+		@endportals = array()
+	}
+	array_push(@endportals, array(name: $endportal, coord: ploc()))
+	store_value('endportals', @endportals)
+	msg(color(LIGHT_PURPLE).$endportal . ' added.')
+	call_alias('/place '.$endportal)
+<<<
+
+nerdch.level.admin:/delendportal $endportal = >>>
+	@endportals = get_value('endportals')
+	if(@endportals == null){
+		@endportals = array()
+	}
+	@found = false
+	foreach(@endportals, @key, @value,
```

Comment:
FYI: foreach(@key: @value in @endportals){} should work on any relatively recent version, and is much cleaner.